src/MarginTrading.SettingsService.Core/Services/IMarketDayOffService.cs

```diff
@@ -9,8 +9,8 @@ namespace MarginTrading.SettingsService.Core.Services
 {
     public interface IMarketDayOffService
     {
-        Task<Dictionary<string, bool>> MarketsStatus(string[] marketIds);
+        Task<Dictionary<string, (DateTime lastTradingDay, bool isTradingEnabled)>> GetMarketsInfo(string[] marketIds);
```

Comment:
Why tuple? You have a contract, let's create domain model for it and use everywhere (maybe I should have done it initially).